aten/src/ATen/native/cpu/Parallel.h

```diff
@@ -7,18 +8,18 @@
 namespace at {
 namespace native {
 
-constexpr size_t _THRESHOLD = 32768;
+constexpr size_t GRAIN_SIZE = 32768;
```

Comment:
Can you please add a comment for this constant?

Reply:
Sure. I'll revise the documentation for the entire setup in another PR as well. In particular the README file.